ModInstances/SpookyNPC.cs

```diff
@@ -315,57 +279,57 @@ public override void GetChat(NPC npc, ref string chat)
                         chat = "I can see it in the headlines: 'Local Hairstylist Murders Patient'.";
                         break;
                 }
-            }
         }
+
         public static int slenderFailsafeTimer;
+
         public override void PostAI(NPC npc)
         {
-            Player player = Main.player[Main.myPlayer];
-            float distanceToStalker = npc.Distance(player.Center);
+            var player = Main.player[Main.myPlayer];
```

Comment:
also not really enjoying all of these swaps from the actual type -> `var`

Reply:
var's are more efficient in this context